src/Model/Behavior/Strategy/Base62Strategy.php

```diff
@@ -7,16 +7,16 @@
 use Tuupola\Base62;
 
 /**
- * Class TuupolaStrategy
+ * Class Base62Strategy
  */
-class TuupolaStrategy implements StrategyInterface
+class Base62Strategy implements StrategyInterface
```

Comment:
Filename needs to be updated to match the classname.

Reply:
Hmm, it looks like the `Base62Strategy` class name matches the filename already

Reply:
I swear the diff showed me the old filename earlier :)  Anyway all good now.